nexpose/nexpose_site.py

```diff
@@ -140,6 +141,11 @@ def AsXML(self, exclude_id):
         xml_alerting = create_element('Alerting')
         xml_data.append(xml_alerting)
 
+        xml_alerts = create_element('Alerting')
```

Comment:
It looks like this element is already being created and appended above this line. Should those lines be removed since this duplicates it?